### PR TITLE
Make docker kibana works with new staging env

### PR DESCRIPTION
**Pull request by peterzhuamazon**
*Issue #, if available:*
P40553676

*Description of changes:*
This PR is to make docker kibana works with new staging env

*Test Results:*
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/497203278

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: kibana/docker/Makefile [hunk 1]
```diff
@@ -24,8 +24,8 @@ BUILD_DATE := $(shell date +'%y.%m.%d')
 
 OPENDISTRO_REGISTRY ?= amazon
 
-ES_VERSION ?= $(shell ../../bin/version-info --es)
-OPENDISTRO_VERSION ?= $(shell ../../bin/version-info --od)
+ES_VERSION ?= $(shell ../../release-tools/scripts/version-info.sh --es)
+OPENDISTRO_VERSION ?= $(shell ../../release-tools/scripts/version-info.sh --od)
 OPENDISTRO_PLUGIN_VERSION ?= $(OPENDISTRO_VERSION).0.zip
 IMAGE_TAG ?= $(OPENDISTRO_REGISTRY)/opendistro-for-elasticsearch-kibana
 ARTIFACTS_URL='https://d3g5vo6xdbdb9a.cloudfront.net/downloads/kibana'
```
Comment (gaiksaya) [inline kibana/docker/Makefile:31]: Do we need this now?

File: release-tools/scripts/docker_download_plugins.sh [hunk 2]
```diff
@@ -5,6 +5,8 @@
 # Requires yq v4.0.0+
 
 set -e
+CATEGORY=$1; echo CATEGORY $CATEGORY; if [ -z "$CATEGORY" ]; then echo "Please enter plugin category as parameter: elasticsearch/kibana"; exit 1; fi
+ARCHTECTURE=$2; echo ARCHITECTURE $ARCHITECTURE
```
Comment (gaiksaya) [inline release-tools/scripts/docker_download_plugins.sh:9]: While running this script in [here](https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/561/files#diff-ad6f24d7ee0e1d39067adb1ccfffd64f8cde0ba89328dd2b329b1afd78c240d7R36) where have we mentioned the architecture?

File: kibana/docker/Makefile (hunk 1 above)
Comment (peterzhuamazon) [reply kibana/docker/Makefile:31]: Yes, the Dockerfile.j2 is parsing these values through Jinja2.

Comment (gaiksaya) [reply kibana/docker/Makefile:31]: I meant `ARTIFACTS_URL='https://d3g5vo6xdbdb9a.cloudfront.net/downloads/kibana'`

File: release-tools/scripts/docker_download_plugins.sh (hunk 2 above)
Comment (peterzhuamazon) [reply release-tools/scripts/docker_download_plugins.sh]: Make changes to the logic a bit.

File: release-tools/scripts/docker_download_plugins.sh [hunk 3]
```diff
@@ -14,32 +16,40 @@ S3_RELEASE_BASEURL=`yq eval '.urls.ODFE.releases' $MANIFEST_FILE`
...
+  if echo ${PLUGINS_ARRAY[$index]} | grep -i reports
+  then
+    plugin_latest=`aws s3api list-objects --bucket $S3_RELEASE_BUCKET --prefix "${PLUGIN_PATH}${OD_VERSION}/${S3_RELEASE_BUILD}/${CATEGORY}-plugins" --query 'Contents[].[Key]' --output text \
+                   | grep -v sha512 | grep ${PLUGINS_ARRAY[$index]} | grep zip | grep linux | grep "$ARCHITECTURE" | sort | tail -n 1`
+  else
+    plugin_latest=`aws s3api list-objects --bucket $S3_RELEASE_BUCKET --prefix "${PLUGIN_PATH}${OD_VERSION}/${S3_RELEASE_BUILD}/${CATEGORY}-plugins" --query 'Contents[].[Key]' --output text \
+                   | grep -v sha512 | grep ${PLUGINS_ARRAY[$index]} | grep zip | sort | tail -n 1`
+  fi
 
   if [ ! -z "$plugin_latest" ]
```
Comment (sreekarjami) [inline release-tools/scripts/docker_download_plugins.sh:49]: We will need another condition to confirm it is a release candidate.

File: kibana/docker/Makefile (hunk 1 above)
Comment (peterzhuamazon) [reply kibana/docker/Makefile:31]: Yes, it is parsed and used in Dockerfile

File: release-tools/scripts/docker_download_plugins.sh (hunk 3 above)
Comment (peterzhuamazon) [reply release-tools/scripts/docker_download_plugins.sh:49]: Like we talked, it is already checked in https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/release-tools/scripts/staging-copy-artifacts.py#L155

**Review by sreekarjami — APPROVED** [review]

**Review by gaiksaya — APPROVED** [review]

**Pull request merged**